docs/examples/fit_NGC6440E.py

```diff
@@ -18,11 +16,14 @@
 parfile = os.path.join(datadir, "NGC6440E.par")
 timfile = os.path.join(datadir, "NGC6440E.tim")
 
-# Define the timing model
-m = mb.get_model(parfile)
+# Read the timing model and the TOAs
+m, t = mb.get_model_and_toas(parfile, timfile)
```

Comment:
It's worth changing the notebooks too, to work this way. If you don't ever use notebooks, you can just make the changes to the `.py` files, they're what's in git anyway, then just doing `make notebooks`.

Reply:
yeah, I agree.  Will try to get that in tomorrow.

Reply:
Do we want to use the new function in all of the notebooks?  Or just tutorial-style ones?

Reply:
Might as well switch over completely unless it's doing something like load TOAs with no model in sight.